lib/sidekiq_fast_debounce.rb

```diff
@@ -80,15 +76,26 @@ module ClassMethods
     # @param delay [#to_f] how many seconds to delay to wait for other jobs
     # @param args [*Array]
     # @return [String] Sidekiq job id
+    def perform_debounce(delay, *args)
+      at = Time.now.to_f + delay.to_f
+      set('debounce' => delay.to_f).perform_at(at, *args)
+    end
+  end
+
+  # Add perform_debounce to Sidekiq::Worker::Setter to enable set(..).perform_debounce(...)
+  module Setter
     def perform_debounce(delay, *args)
       item = {
-        'class' => self,
+        'class' => @klass,
```

Comment:
where is `@klass` coming from? is it Sidekiq internals? a link to a doc or at least Sidekiq code would be nice

Reply:
also is this private stuff or is there a way to expand it with a public interface?

Reply:
`Sidekiq::Worker::Setter` is a class that handles the `set` in `MyWorker.set(queue: :task).perform_async(1234)`